### DISCUSSION TITLE
Getting started with Oqtane on Mac

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#680] [General] [answered]
My designed is working on a custom Oqtane theme and is working on a Mac.  I understand that SQL is required, and I'm wondering if there is a how-t0 guide for getting sql up and running in docker and then configuring oqtane to see that when it starts.

**Comment by mikecasas** [comment, marked as answer]
Can you pull my sql image from Dockerhub?

See if the link below helps. I have a simple how-to.
https://hub.docker.com/repository/docker/mkcsas/oqtane-sql-base

**Reply by markdav-is** [reply]
I'll let @bmckenzie93 know and see if he can get it rolling.    I wonder if he will see that notification...

**Reply by bmckenzie93** [reply]
Let me try and see

**Reply by markdav-is** [reply]
@bmckenzie93 was able to get a debug environment set up on his Mac.

**Reply by nissan** [reply]
Similarly I got ocqtane running on my Mac using  the Microsoft SQL server docker image on dockerhub

**Reply by leigh-pointer** [reply]
I have been exploring using the Mac and VS to develop on Oqtane.  
The first issue I came in to was the file path separator in the ModuleCreator. Using the constant string value of "\\" will break the code as the file path separator for the Mac is "/".

Scanning the Solution for this constant "\\" resulted 7 counts in the Clients and Server projects.

Solution would be to use the .net constant _Path_.**DirectorySeparatorChar** using System.IO
